_schemaV2.graphql

```diff
@@ -10620,6 +10620,30 @@ type HeroUnitLink {
   url: String!
 }
 
+# Hero units rail section
+type HeroUnitsHomeViewSection implements GenericHomeViewSection & Node {
+  # The component that is prescribed for this section
+  component: HomeViewComponent
+  heroUnitsConnection(
+    after: String
+    before: String
+    first: Int
+    last: Int
+
+    # If true will include inactive hero units.
+    private: Boolean = false
+
+    # If present will search by term.
+    term: String
```

Comment:
Shouldn't both be set in Metaphysics instead of being sent by the client)

Reply:
Agree, will do